python/ray/train/v2/_internal/constants.py

```diff
@@ -51,6 +51,12 @@
 # V2 feature flag.
 V2_ENABLED_ENV_VAR = "RAY_TRAIN_V2_ENABLED"
 
+# Environment variables to enable/disable controller/worker structured logging.
+ENABLE_CONTROLLER_LOGGING_ENV_VAR = "RAY_TRAIN_ENABLE_CONTROLLER_LOGGING"
+ENABLE_WORKER_LOGGING_ENV_VAR = "RAY_TRAIN_ENABLE_WORKER_LOGGING"
```

Comment:
@matthewdeng What do you think of this? Controller/worker logs still show up if this env var is set to 0, the current naming might be a bit confusing.

```suggestion
ENABLE_CONTROLLER_LOGGING_ENV_VAR = "RAY_TRAIN_ENABLE_CONTROLLER_STRUCTURED_LOGGING"
ENABLE_WORKER_LOGGING_ENV_VAR = "RAY_TRAIN_ENABLE_WORKER_STRUCTURED_LOGGING"
```

Reply:
Would go even further and include structured in the environment variable
```suggestion
ENABLE_CONTROLLER_STRUCTURED_LOGGING_ENV_VAR = "RAY_TRAIN_ENABLE_CONTROLLER_STRUCTURED_LOGGING"
ENABLE_WORKER_STRUCTURED_LOGGING_ENV_VAR = "RAY_TRAIN_ENABLE_WORKER_STRUCTURED_LOGGING"
```

Reply:
Changed.